whatlies/embedding.py

```diff
@@ -177,7 +177,7 @@ def distance(self, other, metric: str = "cosine"):
 
     def plot(
         self,
-        kind: str = "scatter",
+        kind: str = "arrow",
```

Comment:
This might be a breaking change for some users, so at least they should be clearly notified of this in release notes!
As a side point, I think one of the standards of PRs is to make them as small and relevant as possible and concerned with only one specific feature/issue.

Reply:
You're right. This PR started as a "branch of things to fix for a paper we're writing". It's still a work in progress though. 

You're right about the release notes. I've been adding them to github but it might be time to start adding them to the documentation as well.